scripts/addons/io_scene_gltf2/gltf2_generate.py

```diff
@@ -1028,112 +1010,15 @@ def generate_lights_cmn(operator,
-
-        if glTF.get('extensions') is None:
-            glTF['extensions'] = {}
-
-        extensions = glTF['extensions']
-
-        extensions['KHR_lights_cmn'] = {
-            'lights': lights
-        }
-
-
-def generate_lights_pbr(operator,
-                        context,
-                        export_settings,
-                        glTF):
-    """
-    Generates the top level lights entry.
-    Note: This is currently an experimental feature.
-    """
-
-    lights = []
-
-    #
-    #
-
-    filtered_lights_pbr = export_settings['filtered_lights_pbr']
-
-    for blender_light in filtered_lights_pbr:
-
-        #
-        # Property: light
-        #
-
-        light = {'profile': 'PBR'}
-
-        #
-
-        blender_light_node = None
-
-        for blender_node in blender_light.node_tree.nodes:
-            if isinstance(blender_node, bpy.types.ShaderNodeGroup):
-                if blender_node.node_tree.name.startswith('glTF Directional Light'):
-                    light['type'] = 'directional'
-                    blender_light_node = blender_node
-                    break
-                if blender_node.node_tree.name.startswith('glTF Point Light'):
-                    light['type'] = 'point'
-                    blender_light_node = blender_node
-                    break
-                if blender_node.node_tree.name.startswith('glTF Spot Light'):
-                    light['type'] = 'spot'
-                    blender_light_node = blender_node
-                    break
-
-        if blender_light_node is None:
-            continue
-
-        #
-
-        if light['type'] == 'point' or light['type'] == 'spot':
-
-            positional = {}
-
-            positional['radius'] = blender_light_node.inputs['Radius'].default_value
-
-            light['positional'] = positional
-
-            if light['type'] == 'spot':
-                spot = {}
-
-                angle = blender_light.spot_size * 0.5
-
-                spot['outerAngle'] = angle
-                spot['innerAngle'] = angle - angle * blender_light.spot_blend
-
-                positional['spot'] = spot
-
-        light['color'] = [blender_light_node.inputs['Color'].default_value[0],
-                          blender_light_node.inputs['Color'].default_value[1],
-                          blender_light_node.inputs['Color'].default_value[2]]
-        light['strength'] = blender_light_node.inputs['Strength'].default_value
-
-        #
-
-        light['name'] = blender_light.name
-
-        #
-        #
-
-        lights.append(light)
-
-    #
-    #
-
-    if len(lights) > 0:
-        create_extensionsUsed(operator, context, export_settings, glTF, 'KHR_lights_pbr')
-        create_extensionsRequired(operator, context, export_settings, glTF, 'KHR_lights_pbr')
+        create_extensionsUsed(operator, context, export_settings, glTF, 'KHR_lights')
+        create_extensionsRequired(operator, context, export_settings, glTF, 'KHR_lights')
```

Comment:
I think I'd vote to omit `KHR_lights` from `extensionsRequired`. Any concerns with that?